pippo-core/src/main/java/ro/pippo/core/route/UrlResourceHandler.java

```diff
@@ -33,12 +35,20 @@
 
     private static final Logger log = LoggerFactory.getLogger(UrlResourceHandler.class);
 
+    private static final Pattern VERSION_PATTERN = Pattern.compile("-[0-9]+\\.");
```

Comment:
Maybe instead of using `-` which is a fairly common character, perhaps we should use a different delimiter like `~` or another alternative.